src/App.js

```diff
@@ -20,27 +30,38 @@ const DashboardScreen = imported(() => import('screens/DashboardScreen'));
 function App() {
   const currentSession = reactLocalStorage.getObject('currentSession');
 
-  const [{ data: myProfile, loading, error }] = useAxios(
-    `/users/${currentSession.id}`
-  );
-
-  if (loading) {
-    // we should show some loading screen maybe
-    return <div />;
-  }
-
-  if (error) {
+  if (currentSession) {
+    AuthService.setCurrentSession(currentSession);
+  } else {
     return window.location.replace(`/`);
   }
 
-  if (myProfile) {
-    AuthService.setCurrentSession(myProfile);
-  }
+  // const [{ data: myProfile, loading, error }] = useAxios(
+  //   `/users/${currentSession.id}`
+  // );
+
+  // if (loading) {
+  //   return (
+  //     <div className="w-full h-full flex justify-center items-center">
+  //       <Spin size="large" />
+  //     </div>
+  //   );
+  // }
+
+  // if (error) {
+  //   return window.location.replace(`/`);
+  // }
+
+  // if (myProfile) {
+  //   AuthService.setCurrentSession(myProfile);
+  // }
 
   return (
     <Router>
       <Switch>
-        <Route exact path="/" component={HomePage} />
+        <Redirect exact from="/" to="/student" />
```

Comment:
i don't understand this, don't redirect to student, both /student and / should render homescreen
your code mean redirect to student on `/` which doesn't make much sense